docs/source/getting-started.rst

```diff
@@ -42,7 +42,7 @@ To execute a gauge-python project, run the following command in the project:
 
         $ gauge specs/
 
-    .. _Gauge documentation: http://getgauge.io/documentation/user/current/
+    .. _Gauge documentation: https://docs.getgauge.io/
```

Comment:
@uniconstructor thanks for pointing this out
The new home for documentation is 
https://docs.gauge.org

Can you modify this pull request to the the new location?

Reply:
@zabil Sorry, but I don't understand what do I need to do. :)
What means "modify this pull request to the new location"?
What is "location" of the pull request? Am I made a pull request to incorrect repository - or this is about something else?

Reply:
Sorry about that! I meant modify this pull request and change

> https://docs.getgauge.io/

to 

> https://docs.gauge.org/